preprocessors/openstreetmap/osm_service.py

```diff
@@ -1008,7 +1018,7 @@ def get_coordinates(content):
     place_response = requests.get(request).json()
 
     if not check_google_response(place_response):
-        return None
+        return jsonify("Processing Aborted!"), 400
```

Comment:
This is not a client error.

Reply:
Please improve the debugging message here as well. "Processing Aborted" does not indicate the cause.